modules/selenium/build.gradle

```diff
@@ -3,8 +3,11 @@ description = "Testcontainers :: Selenium"
 dependencies {
     compile project(':testcontainers')
 
-    provided 'org.seleniumhq.selenium:selenium-remote-driver:3.141.59'
-    provided 'org.seleniumhq.selenium:selenium-chrome-driver:3.141.59'
+    compileOnly 'org.seleniumhq.selenium:selenium-remote-driver:3.141.59'
+    compileOnly 'org.seleniumhq.selenium:selenium-chrome-driver:3.141.59'
+    runtimeOnly 'org.seleniumhq.selenium:selenium-remote-driver:3.141.59'
+    runtimeOnly 'org.seleniumhq.selenium:selenium-chrome-driver:3.141.59'
```

Comment:
@bsideup re your previous comment:
https://github.com/testcontainers/testcontainers-java/pull/3347#discussion_r571458479

you were right; we were ending up with a different POM, which was no good.

I could be missing a convenient way, but I believe that `compileOnly + runtimeOnly` achieves the same effect as `provided` did.

I've verified that we're getting identical POMs:

```
# for each of master branch and this branch
$ rm -rf ~/.m2/org/testcontainers
$ ./gradlew publishToMavenLocal

$ mv ~/.m2/org/testcontainers testcontainers-$THEBRANCH
$ cd testcontainers-$THEBRANCH

# discard any non-pom files
$ find . -type f -not -name '*.pom' | xargs rm

# now, with the two directories side-by-side containing POMs generated on each branch:
$ diff -s -r testcontainers-master/ testcontainers-gradlew-update-6.7/
```

The result is:
```
Files testcontainers-master/cassandra/unspecified/cassandra-unspecified.pom and testcontainers-gradlew-update-6.7/cassandra/unspecified/cassandra-unspecified.pom are identical
Files testcontainers-master/clickhouse/unspecified/clickhouse-unspecified.pom and testcontainers-gradlew-update-6.7/clickhouse/unspecified/clickhouse-unspecified.pom are identical
Files testcontainers-master/cockroachdb/unspecified/cockroachdb-unspecified.pom and testcontainers-gradlew-update-6.7/cockroachdb/unspecified/cockroachdb-unspecified.pom are identical
Files testcontainers-master/couchbase/unspecified/couchbase-unspecified.pom and testcontainers-gradlew-update-6.7/couchbase/unspecified/couchbase-unspecified.pom are identical
Files testcontainers-master/database-commons/unspecified/database-commons-unspecified.pom and testcontainers-gradlew-update-6.7/database-commons/unspecified/database-commons-unspecified.pom are identical
Files testcontainers-master/db2/unspecified/db2-unspecified.pom and testcontainers-gradlew-update-6.7/db2/unspecified/db2-unspecified.pom are identical
Files testcontainers-master/dynalite/unspecified/dynalite-unspecified.pom and testcontainers-gradlew-update-6.7/dynalite/unspecified/dynalite-unspecified.pom are identical
Files testcontainers-master/elasticsearch/unspecified/elasticsearch-unspecified.pom and testcontainers-gradlew-update-6.7/elasticsearch/unspecified/elasticsearch-unspecified.pom are identical
Files testcontainers-master/gcloud/unspecified/gcloud-unspecified.pom and testcontainers-gradlew-update-6.7/gcloud/unspecified/gcloud-unspecified.pom are identical
Files testcontainers-master/influxdb/unspecified/influxdb-unspecified.pom and testcontainers-gradlew-update-6.7/influxdb/unspecified/influxdb-unspecified.pom are identical
Files testcontainers-master/jdbc/unspecified/jdbc-unspecified.pom and testcontainers-gradlew-update-6.7/jdbc/unspecified/jdbc-unspecified.pom are identical
Files testcontainers-master/junit-jupiter/unspecified/junit-jupiter-unspecified.pom and testcontainers-gradlew-update-6.7/junit-jupiter/unspecified/junit-jupiter-unspecified.pom are identical
Files testcontainers-master/kafka/unspecified/kafka-unspecified.pom and testcontainers-gradlew-update-6.7/kafka/unspecified/kafka-unspecified.pom are identical
Files testcontainers-master/localstack/unspecified/localstack-unspecified.pom and testcontainers-gradlew-update-6.7/localstack/unspecified/localstack-unspecified.pom are identical
Files testcontainers-master/mariadb/unspecified/mariadb-unspecified.pom and testcontainers-gradlew-update-6.7/mariadb/unspecified/mariadb-unspecified.pom are identical
Files testcontainers-master/mockserver/unspecified/mockserver-unspecified.pom and testcontainers-gradlew-update-6.7/mockserver/unspecified/mockserver-unspecified.pom are identical
Files testcontainers-master/mongodb/unspecified/mongodb-unspecified.pom and testcontainers-gradlew-update-6.7/mongodb/unspecified/mongodb-unspecified.pom are identical
Files testcontainers-master/mssqlserver/unspecified/mssqlserver-unspecified.pom and testcontainers-gradlew-update-6.7/mssqlserver/unspecified/mssqlserver-unspecified.pom are identical
Files testcontainers-master/mysql/unspecified/mysql-unspecified.pom and testcontainers-gradlew-update-6.7/mysql/unspecified/mysql-unspecified.pom are identical
Files testcontainers-master/neo4j/unspecified/neo4j-unspecified.pom and testcontainers-gradlew-update-6.7/neo4j/unspecified/neo4j-unspecified.pom are identical
Files testcontainers-master/nginx/unspecified/nginx-unspecified.pom and testcontainers-gradlew-update-6.7/nginx/unspecified/nginx-unspecified.pom are identical
Files testcontainers-master/oracle-xe/unspecified/oracle-xe-unspecified.pom and testcontainers-gradlew-update-6.7/oracle-xe/unspecified/oracle-xe-unspecified.pom are identical
Files testcontainers-master/orientdb/unspecified/orientdb-unspecified.pom and testcontainers-gradlew-update-6.7/orientdb/unspecified/orientdb-unspecified.pom are identical
Files testcontainers-master/postgresql/unspecified/postgresql-unspecified.pom and testcontainers-gradlew-update-6.7/postgresql/unspecified/postgresql-unspecified.pom are identical
Files testcontainers-master/presto/unspecified/presto-unspecified.pom and testcontainers-gradlew-update-6.7/presto/unspecified/presto-unspecified.pom are identical
Files testcontainers-master/pulsar/unspecified/pulsar-unspecified.pom and testcontainers-gradlew-update-6.7/pulsar/unspecified/pulsar-unspecified.pom are identical
Files testcontainers-master/r2dbc/unspecified/r2dbc-unspecified.pom and testcontainers-gradlew-update-6.7/r2dbc/unspecified/r2dbc-unspecified.pom are identical
Files testcontainers-master/rabbitmq/unspecified/rabbitmq-unspecified.pom and testcontainers-gradlew-update-6.7/rabbitmq/unspecified/rabbitmq-unspecified.pom are identical
Files testcontainers-master/selenium/unspecified/selenium-unspecified.pom and testcontainers-gradlew-update-6.7/selenium/unspecified/selenium-unspecified.pom are identical
Files testcontainers-master/solr/unspecified/solr-unspecified.pom and testcontainers-gradlew-update-6.7/solr/unspecified/solr-unspecified.pom are identical
Files testcontainers-master/spock/unspecified/spock-unspecified.pom and testcontainers-gradlew-update-6.7/spock/unspecified/spock-unspecified.pom are identical
Files testcontainers-master/testcontainers/unspecified/testcontainers-unspecified.pom and testcontainers-gradlew-update-6.7/testcontainers/unspecified/testcontainers-unspecified.pom are identical
Files testcontainers-master/testcontainers-bom/unspecified/testcontainers-bom-unspecified.pom and testcontainers-gradlew-update-6.7/testcontainers-bom/unspecified/testcontainers-bom-unspecified.pom are identical
Files testcontainers-master/toxiproxy/unspecified/toxiproxy-unspecified.pom and testcontainers-gradlew-update-6.7/toxiproxy/unspecified/toxiproxy-unspecified.pom are identical
Files testcontainers-master/vault/unspecified/vault-unspecified.pom and testcontainers-gradlew-update-6.7/vault/unspecified/vault-unspecified.pom are identical
```

Reply:
FWIW I consider `compileOnly + runtimeOnly` to be functional but ugly. If you know of a reason why it's not actually functional, or know of a less ugly way, it'd be awesome to know 😄